### PR TITLE
fix: Context structure

### PR DESCRIPTION
**Pull request by vahidlazio**

### CODE REVIEW

File: pkg/confidence/confidence.go [hunk 1]
```diff
@@ -83,10 +83,14 @@ func (e Confidence) PutContext(key string, value interface{}) {
 	e.contextMap[key] = value
 }
 
-func (e Confidence) Track(ctx context.Context, eventName string, message map[string]interface{}) *sync.WaitGroup {
-	newMap := e.GetContext()
+func (e Confidence) Track(ctx context.Context, eventName string, data map[string]interface{}) *sync.WaitGroup {
```
Comment (fabriziodemaria) [inline pkg/confidence/confidence.go:86]: nit: eventName should be the first argument

**Review by fabriziodemaria — APPROVED** [review]
Would be nice with a test, but nothing blocking

File: pkg/confidence/confidence.go (hunk 1 above)
Comment (vahidlazio) [reply pkg/confidence/confidence.go:86]: the first context is coming from go, it's not confidence context. i think it's pretty standard to have the go context as the first parameter. you still think we should move it?

Comment (fabriziodemaria) [reply pkg/confidence/confidence.go:90]: Ah you are right, no probably makes sense to keep it there then

**Review by fabriziodemaria — APPROVED** [review]

**Pull request merged**